### DISCUSSION TITLE
"Software Boundary" - inconsistent types of devices and term MAY is confusing

### DISCUSSION THREAD

**Discussion by dschien** [Green-Software-Foundation/sci#181] [SCI feedback]
The list is a mix of types of devices and service functions.

Additionally, the term "MAY" is confusing - is this supposed to be an exhaustive list? I suppose not - so it should be explicitly stated that these are examples.

This section requires review and generalisation, so it can support a standard.

**Comment by atg-abhishek** [comment]
Thanks for sharing that @dschien - please feel free to make a PR with your suggested changes and we can review there.
